scenariolib/event_pageview.go

```diff
@@ -63,17 +70,22 @@ func newViewEvent(e *JSONEvent, c *Config) (*ViewEvent, error) {
 // Execute the view event, sending a view event to the usage analytics
 func (ve *ViewEvent) Execute(v *Visit) error {
 	if v.LastResponse == nil {
-		return errors.New("LastResponse was nil, cannot send a pageview. Please use a search event before.")
+		return errors.New("No query before pageView event, use a search event first")
```

Comment:
In theory, view events are not required to be linked to a search event.

Reply:
Same as above.

And there is a parameter on a search event to ignore the search event, so it looks like only a view has happened.